TLM/TLM/Util/Record/LaneConnectionRecord.cs

```diff
@@ -103,7 +103,7 @@ public class LaneConnectionRecord : IRecordable {
                     var laneData = new LaneConnectionRecord {
                         LaneId = laneIdAndIndex.laneId,
                         LaneIndex = (byte)laneIdAndIndex.laneIndex,
-                        StartNode = (bool)extSegmentManager.IsStartNode(segmentId, nodeId),
+                        StartNode = netSegment.IsStartnode(nodeId),
```

Comment:
`NetSegment.IsStartnode` capitalisation, should be `IsStartNode`

Reply:
The `IsStartnode` (lowercase `n`) always returns a bool so I've been using that in cases where there were no `null` checks (the `IsStartNode` - capital `N` - version of the extension will return `null` if the node does not belong to segment).

Reply:
BTW, I didn't add the `IsStartnode` extension, it was already there. I moved both extensions next to each other in recent PR.

Maybe the `IsStartNode` (one that checks node belongs to segment) should be renamed `IsValidStartNode`? Then the other one can be `IsStartNode` with correct capitalisation. Or something along those lines...? If we want to rename the methods this PR is a good place to do it.

Reply:
So its ok to rename 2500000 other locations in the code, but it makes people hesitant whether the bad naming should be fixed or not? 😀 of course, rename it to something better.

Reply:
done